lib/api/gitlab.js

```diff
@@ -64,6 +64,15 @@ async function initRepo(repoName, token, endpoint) {
   if (endpoint) {
     process.env.GITLAB_ENDPOINT = endpoint;
   }
+  try {
+    logger.debug(`Determining Gitlab API version`);
+    // projects/visible route deprecated in v4
+    await glGot(`projects/visible`);
```

Comment:
My only concern about this particular end-point is that it could return a massive amount of data.

The only end-point that may result in less overhead would be `/projects/owned`, which was also removed in v4.

I kind of wish the `/version` [end-point](https://docs.gitlab.com/ee/api/version.html) would just echo back its version.